src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -135,13 +137,17 @@
         @MergingIsManual
         protected IntervalList intervals;
 
-        /** The count of sites with a given observed depth. */
+        /** The histogram of coverages. Excludes bases with quality below MINIMUM_BASE_QUALITY */
```

Comment:
I like the "count of sites" verbiage. it clearly a histogram...but saying "count of sites" makes it clear that we are tallying the number of sites at a given depth...I would modify the original by adding "depth here is calculated excluding ...."